jobs/sentry-error-loader/sentry_error_loader.py

```diff
@@ -114,8 +116,9 @@ def main(
 ):
     target_date = pendulum.instance(logical_date).date()
     extract = fetch_and_clean_from_clickhouse(project_slug, target_date)
-    fs = get_fs()
-    return extract.save_to_gcs(fs=fs)
+    if not extract.data.empty:
+        fs = get_fs()
+        return extract.save_to_gcs(fs=fs)
```

Comment:
can you log that we aren't saving a file because it's empty?

Reply:
Addressed in [8650185](https://github.com/cal-itp/data-infra/pull/2356/commits/86501859f71672b832bbab6f639a22064c08479d)